GETTING_STARTED.md

```diff
@@ -79,6 +80,15 @@ show_result(img, result, model.CLASSES)
 imgs = ['test1.jpg', 'test2.jpg']
 for i, result in enumerate(inference_detector(model, imgs)):
     show_result(imgs[i], result, model.CLASSES, out_file='result_{}.jpg'.format(i))
+
+# test a video and show the results
+cap = mmcv.VideoReader('video.mp4')
+while True:
+    frame = cap.read()
+    if frame is None:
+        break
+    result = inference_detector(model, frame)
+    show_result(frame, result, model.CLASSES, wait_time=1)
```

Comment:
It can be like this:
```python
video = mmcv.VideoReader('video.mp4')
for frame in video:
    result = inference_detector(model, frame)
    show_result(frame, result, model.CLASSES, wait_time=1)
```

Reply:
Sorry, mmcv is unfamiliar to me.
I had modified